config/webpack.config.js

```diff
@@ -121,7 +121,10 @@ module.exports = {
 		outputStyle: 'expanded'
 	},
 	postcss: () => {
-		return [ autoprefixer({ browsers: ['> 1%', 'last 2 versions', 'ie >= 8', 'ff ESR', 'bb >= 7', 'iOS >= 5'] }) ];
+		return [ autoprefixer({
+			browsers: ['> 1%', 'last 2 versions', 'ie >= 8', 'ff ESR', 'bb >= 7', 'iOS >= 5'],
```

Comment:
Hmm, iOS >= 6 maybe too? Would get rid of dodgy old gradient syntax I believe =]